cmd/logger/message/log/entry.go

```diff
@@ -26,7 +26,7 @@ type Args struct {
 // Trace - defines the trace.
 type Trace struct {
 	Message   string            `json:"message,omitempty"`
-	Source    []string          `json:"source,omitempty"`
+	Source    []string          `json:"source,omitempty" shadow:"omit"`
```

Comment:
This again defeats the purpose, we can't make it generic without bringing in all these new annotations. It may be easier than we have a key-value type of association sensitive function. Rather than a list of strings, we should have a bit more context. Such that we can redact the right kind of values. Let's say we are omitting Source here but we don't know why i.e there is no way to dynamically disable it.